### PR TITLE
V3 pipu

### PR DESCRIPTION
**Pull request by pipu**

### CODE REVIEW

File: cocos/editor-support/cocostudio/Android.mk [hunk 1]
```diff
@@ -58,13 +58,29 @@ ActionTimeline/CCNodeReader.cpp \
...
+../../../external/protocolBuf/src/google/protobuf/message_lite.cc \
+../../../external/protocolBuf/src/google/protobuf/repeated_field.cc \
+../../../external/protocolBuf/src/google/protobuf/wire_format_lite.cc \
+../../../external/protocolBuf/src/google/protobuf/io/coded_stream.cc \
+../../../external/protocolBuf/src/google/protobuf/io/zero_copy_stream.cc \
+../../../external/protocolBuf/src/google/protobuf/io/zero_copy_stream_impl.cc \
+../../../external/protocolBuf/src/google/protobuf/io/zero_copy_stream_impl_lite.cc \
+../../../external/protocolBuf/src/google/protobuf/stubs/common.cc \
+../../../external/protocolBuf/src/google/protobuf/stubs/once.cc \
+../../../external/protocolBuf/src/google/protobuf/stubs/stringprintf.cc \
```
Comment (minggo) [inline cocos/editor-support/cocostudio/Android.mk]: You should link protocol buffer, not include these files in your project.

**Comment by pipu** [comment]
Copy that

**Comment by minggo** [comment]
And please continue modification in a pull request.
Don't send a new one when do some modification.
Or all comments will lost.

**Comment by minggo** [comment]
Please don't add files into `cocos/scripting/lua-bindings/script/cocostudio`, now they are named `cocos/scripting/lua-bindings/script/cocosstudio`.

**Comment by darkdukey** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
[ci rebuild]

**Comment by zilongshanren** [comment]
[ci rebuild]

**Pull request merged**